src/Compilers/CSharp/Test/Emit/Attributes/EmitTestStrongNameProvider.cs

```diff
@@ -1,5 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
+#if NET461
```

Comment:
I'm a bit conflicted about this approach. There are some test classes that simply won't run on CoreClr at all. I could mark every test in there as desktop only. Or this approach. Open to suggestion here.

Reply:
Ah, hadn't noticed this comment when I opened in codeflow. As I mentioned above, I dislike this approach as it's nondiscoverable later. I'd say we should just mark every test.

---
In reply to: [170481557](https://github.com/dotnet/roslyn/pull/25050#discussion_r170481557) [](ancestors = 170481557)